### PR TITLE
Remove emit-metrics flag support and update jobs

### PR DESCRIPTION
**Pull request by chizhg**
**What this PR does, why we need it**:
Fix https://github.com/knative/test-infra/issues/1568

Remove the support for emit-metrics flag and update all periodic jobs

/cc @adrcunha 
/cc @chaodaiG

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -170,8 +168,6 @@ This is a helper script for Knative E2E test scripts. To use it:
    cluster. You can also use the following boolean (0 is false, 1 is true)
```
Comment (adrcunha) [inline scripts/README.md]: Remove the last sentence of this paragraph.

File: scripts/README.md [hunk 2]
```diff
@@ -170,8 +168,6 @@ This is a helper script for Knative E2E test scripts. To use it:
...
    environment variables for the logic:
 
-   - `EMIT_METRICS`: true if `--emit-metrics` was passed.
-
    All environment variables above are marked read-only.
```
Comment (adrcunha) [inline scripts/README.md]: Remove as well.

File: scripts/README.md (hunk 1 above)
Comment (chizhg) [reply scripts/README.md]: Done.

File: scripts/README.md (hunk 2 above)
Comment (chizhg) [reply scripts/README.md]: Done.

**Comment by adrcunha** [comment]
Thanks!

/approve
/lgtm

**Pull request merged**